scripts/zones/Apollyon/mobs/Dee_Wapa_the_Desolator.lua

```diff
@@ -2,52 +2,81 @@
 -- Area: Apollyon CS
 --  Mob: Dee Wapa the Desolator
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+local ID = require("scripts/zones/Apollyon/IDs")
 
 function onMobSpawn(mob)
```

Comment:
ffxiclopedia is claiming each of these bosses have different damage type weaknesses and immunities. I'll admit I'm too lazy to check pools, but are these being handled?

Reply:
I think they're missing their weakness/resistance since I wasn't sure what values to use.